src/commands/module/add.ts

```diff
@@ -248,3 +253,19 @@ async function resolveModule(
     pkgVersion,
   }
 }
+
+function getRegistryFromNpmrc() {
+  const npmrcPath = join(homedir(), '.npmrc')
```

Comment:
should we also support the current directory?

Reply:
sure, sir, support added